lib/topological_inventory/ansible_tower/parser/service_offering_node.rb

```diff
@@ -26,6 +27,16 @@ def parse_service_offering_node(node)
             }
           )
         )
+
+        credentials = offering_node[:credentials].to_a
+        if credentials.present?
```

Comment:
same like ^^^, `present?` is not necessary

Reply:
done 👍